music_album.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item_music'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :release_year, :artist
+
+  def initialize(title, artist, release_year)
+    super(title)
+    @artist = artist
+    @release_year = release_year
+    @on_spotify = false
+  end
+
+  def can_be_archived?
+    super && @on_spotify
```

Comment:
MusicAlbum should extend Items class that was already created and has the method that should be overriden.